templates/hub-resources.yaml

```diff
@@ -54,4 +54,77 @@ metadata:
   name: default
 supportedRegions:
 - local
+---
+apiVersion: api.acorn.io/v1
+kind: App
+metadata:
+  name: acorn-istio-plugin
+  namespace: acorn
+spec:
+  image: ghcr.io/acorn-io/acorn-istio-plugin:main
```

Comment:
Do we want to change this to prod instead? I don't remember if that's what we planned on doing.

Reply:
Ah, that is what we planned on doing. Good catch!